.vscode/launch.json

```diff
@@ -0,0 +1,61 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
```

Comment:
I use VSCode, so I wouldn't mind :)

Reply:
@aaronreynoza The .vscode folders should not have an effect on any other IDE's that is out there so I think it would be harmless. 

I understand that it might not be useful if you are not using VSCode but for developers that do it would assist greatly, with minimal impact on other IDE's.

I will bring this up in standup and revert these files and associated changes if we deem it should not be part of the repo.